src/Routes/RosPage/RosPage.js

```diff
@@ -125,7 +124,7 @@ class RosPage extends React.Component {
         fetchParams?.stateFilter?.forEach((stateFilterValue) => {
             query.append('state', stateFilterValue);
         });
-        fetchParams?.osFilter?.forEach((osFilterValue) => {
+        fetchParams?.filters?.osFilter?.forEach((osFilterValue) => {
```

Comment:
@r14chandra I am curious to know that in case the frontend sends the version not supported by ROS, how the backend will respond to that with your latest backend changes?

Reply:
In case of unsupported versions, the backend will send no system as there would not be any systems present in the DB with that OS version.